### PR TITLE
Register http confidant from majordomo.

### PR DESCRIPTION
**Pull request by catwith1hat**
I found it useful to keep secrets off-host and have dirk fetch those on 
startup via https request. Therefore I'd like to add http from majordomo.

### CODE REVIEW

File: util/majordomo.go [hunk 1]
```diff
@@ -56,6 +57,16 @@ func InitMajordomo(ctx context.Context) (majordomo.Service, error) {
 		return nil, errors.Wrap(err, "failed to register file confidant")
 	}
 
+	httpConfidant, err := httpconfidant.New(ctx,
+		httpconfidant.WithLogLevel(LogLevel("majordomo.confidants.http")),
+	)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create file confidant")
```
Comment (mcdee) [inline util/majordomo.go:64]: Please could you change this error message to http confidant?  Thanks.

**Review by mcdee — CHANGES_REQUESTED** [review]
Thanks for this PR.  One minor tweak requested.

File: util/majordomo.go (hunk 1 above)
Comment (catwith1hat) [reply util/majordomo.go]: Fix. Sorry, I overlooked that copy&paste error.

**Review by mcdee — APPROVED** [review]

File: util/majordomo.go (hunk 1 above)
Comment (mcdee) [reply util/majordomo.go]: Thanks!

**Pull request merged**